internal/guest/runtime/hcsv2/uvm.go

```diff
@@ -582,11 +601,18 @@ func (h *Host) ExecProcess(ctx context.Context, containerID string, params prot.
 		} else {
 			// Windows uses a different field for command, there's no enforcement
 			// around this yet for Windows so this is Linux specific at the moment.
-			envToKeep, err = h.securityPolicyEnforcer.EnforceExecInContainerPolicy(containerID, params.OCIProcess.Args, params.OCIProcess.Env, params.OCIProcess.Cwd)
+			var allowStdioAccess bool
```

Comment:
minor nit: the `var allowStdioAccess bool` separates the above comment from the line it applies to (`EnforceExecInContainerPolicy`)